ovoscope/diff.py

```
@@ -0,0 +1,255 @@
+    """
+    with open(path, "r", encoding="utf-8") as fh:
+        payload = json.load(fh)
+    return payload.get("expected_messages", [])
+
+
+def diff_fixtures(
+    expected_path: str,
+    actual_path: str,
+    *,
+    ignore_context: bool = True,
+    strict_order: bool = True,
```

Ruff (0.15.5)

[warning] Unused function argument: `strict_order`

(ARG001)